From 514e0bcff568d1da5faa997125535adcf102927e Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Rapha=C3=ABl=20Hertzog?= <hertzog@debian.org>
Date: Mon, 28 Feb 2011 21:41:35 +0100
Subject: [PATCH 1/3] Split the Developer's Duties chapter in two

One part on the technical work and one part on the administrative side.

Put the technical work first because that's the most interesting part
for the reader.
---
 developer-duties.dbk |   17 +++++++++++++++++
 1 files changed, 17 insertions(+), 0 deletions(-)

diff --git a/developer-duties.dbk b/developer-duties.dbk
index f09e3cf..005350a 100644
--- a/developer-duties.dbk
+++ b/developer-duties.dbk
@@ -5,6 +5,22 @@
 ]>
 <chapter id="developer-duties">
 <title>Debian Developer's Duties</title>
+
+<section id="package-maintainer-duties">
+<title>Package Maintainer's Duties</title>
+<para>As a package maintainer, you're supposed to fulfill the Debian
+Social Contract by providing high-quality packages that are well integrated
+in the system and that adhere to the Debian Policy.</para>
+
+
+</section>
+
+<section id="administrative-duties">
+<title>Administrative Duties</title>
+<para>A project of the size of Debian relies on some administrative
+infrastructure to keep track of everything. As a project member, you
+have some duties to ensure everything keeps running smoothly.</para>
+
 <section id="user-maint">
 <title>Maintaining your Debian information</title>
 <para>
@@ -214,6 +230,7 @@ RT' somewhere in the subject line (case doesn't matter).
 </listitem>
 </orderedlist>
 </section>
+</section>
 
 </chapter>
 
-- 
1.7.4.1

